core/src/main/java/com/xmartlabs/bigbang/core/module/OkHttpModule.kt

```diff
@@ -73,13 +73,9 @@ open class OkHttpModule {
 
     try {
       val statFs = StatFs(dir.absolutePath)
-      val blockCount: Long
-      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
-        blockCount = statFs.blockCountLong
-      } else {
-        @Suppress("DEPRECATION")
-        blockCount = statFs.blockCount.toLong()
-      }
+      @Suppress("DEPRECATION")
+      val blockCount = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2)
```

Comment:
because I don't like a function in this case, something like `isSdkVersionGreaterThanJellyBeanRm` in my opinion this is not ok, it should be more generic, it shouldn't be in OkHttpModule. We should have something like= `contex.version.isSdkVersionAfter(Build.VERSION_CODES.JELLY_BEAN_MR2)`